VAMobile/src/screens/HealthScreen/SecureMessaging/SecureMessaging.test.tsx

```diff
@@ -17,6 +19,30 @@ jest.mock('store/slices', () => {
         payload: '',
       }
     }),
+    resetSaveDraftComplete: jest.fn(() => {
```

Comment:
Had to update this so the unit tests worked once I deleted the claims and appeals slice. no idea why it was working before deletion, but whatever lol